tests/RelationsTest.php

```diff
@@ -61,24 +61,24 @@ public function testBelongsTo(): void
     public function testHasOne(): void
     {
         $user = User::create(['name' => 'John Doe']);
-        Role::create(['type' => 'admin', 'user_id' => $user->_id]);
+        Role::create(['type' => 'admin', 'roleuser_id' => $user->_id]);
```

Comment:
What happens if we leave name `user_id`?

Reply:
Well, if we leave 'user_id' then there is no way to check that **HasOne / HasMany** respects **$localKey** parameter.

Reply:
I will make a cleaner test, using a different Model to test this.